app/src/main/java/com/framgia/soundclound/data/source/remote/TrackRemoteDataSource.java

```diff
@@ -98,6 +99,10 @@ public void onComplete() {
                     cursor.getInt(cursor.getColumnIndex(MediaStore.Audio.Media.DURATION)));
             track.setPublisherMetadata(new PublisherMetadata(cursor.getString(
                     cursor.getColumnIndex(MediaStore.Audio.Media.ARTIST))));
+            if (cursor.getColumnIndex(MediaStore.Audio.AlbumColumns.ALBUM_ART) > 0) {
+                track.setArtworkUrl
+                        (cursor.getString(cursor.getColumnIndex(MediaStore.Audio.AlbumColumns.ALBUM_ART)));
```

Comment:
ERROR: 'method call lparen' have incorrect indentation level 24, expected level should be 16.